src/interner.rs

```diff
@@ -0,0 +1,10 @@
+//! A string interner.
```

Comment:
I guess this is unintended :slightly_smiling_face:

Reply:
For sure!